linters/sqlfluff/test/sqlfluff_test.ts

```diff
@@ -0,0 +1,17 @@
+import * as fs from "fs";
+import path from "path";
+import { defaultLinterCheckTest, defaultLinterFmtTest, TestCallback } from "tests";
+
+defaultLinterCheckTest({linterName: "sqlfluff", namedTestPrefixes: ["basic_check"]});
+
+// Due to sqlfluff's fix subcommand being disabled by default, we need to manually enable it in our test's
+// trunk.yaml.
+const fmtCallbacks: TestCallback = (driver) => {
+    const trunkYamlPath = path.resolve(driver.sandboxPath ?? "", ".trunk/trunk.yaml");
+    const currentContents = fs.readFileSync(trunkYamlPath, "utf8");
+    const sqlfluffRegex = /- sqlfluff@((\d.?)+)\n/;
+    const newContents = currentContents.replace(sqlfluffRegex, "- sqlfluff@$1:\n        commands: [lint, fix]\n");
+    fs.writeFileSync(trunkYamlPath, newContents);
+};
+
+defaultLinterFmtTest({linterName: "sqlfluff", namedTestPrefixes: ["basic_fmt"], preCheck: fmtCallbacks});
```

Comment:
Eventually I'd like to add a demo of cases where sqlfluff fmt does bad things, but for now this is fine.